lib/Twig/Cache/Null.php

```diff
@@ -45,6 +47,9 @@ public function write($key, $content)
      */
     public function load($key)
     {
+        eval('?>'.$this->buffer[$key]);
```

Comment:
Twig does prevent loading the same template twice (same being a template with the same class name). So, I think this code is safe, except if you are doing very weird things I suppose.